src/config/serverConfigDefaults.ts

```diff
@@ -44,6 +44,7 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     skin_authorization_message:
         'Access to this portal is only available to authorized users.',
     skin_documentation_about: 'About-Us.md',
+    skin_documentation_software: 'Software-Acknowledgments.md',
```

Comment:
Nice US spelling of acknowledgment

FYI - If we want to allow people to update this config using portal.properties it will also need to be added to config_service.jsp in the backend. We already allow people to override the footer in its entirety, so seems overkill to add it there but just so you know